lib/animina/narratives/resources/story.ex

```diff
@@ -172,4 +183,21 @@ defmodule Animina.Narratives.Story do
       reference :user, on_delete: :delete
     end
   end
+
+  defp update_user_registration_completed_at(user_id) do
+    user = Accounts.User.by_id!(user_id)
+
+    case by_user_id(user_id) do
+      {:ok, stories} ->
+        if Enum.count(stories) >= @number_of_stories_required_for_complete_registration &&
+             user.registration_completed_at == nil do
+          Accounts.User.update(user, %{registration_completed_at: DateTime.utc_now()})
+        else
+          Accounts.User.update(user, %{registration_completed_at: nil})
```

Comment:
Why would one ever update `registration_completed_at` to `nil`? That attribute is `nil` by default and can only be updated once with a timestamp. After that it is set in stone.

Reply:
@wintermeyer , kindly provide clarity on what should happen if I delete all my stories